var/spack/repos/builtin/packages/py-amrex/package.py

```diff
@@ -60,10 +65,10 @@ class PyAmrex(CMakePackage, PythonExtension, CudaPackage, ROCmPackage):
 
     depends_on("cxx", type="build")
 
-    depends_on("cmake@3.20:", type="build")
+    depends_on("cmake@3.24:", type="build")
     depends_on("python@3.8:", type=("build", "run"))
     depends_on("py-mpi4py@2.1.0:", type=("build", "run"), when="+mpi")
-    depends_on("py-numpy@1.15.0:1", type=("build", "run"))
+    depends_on("py-numpy@1.15.0:", type=("build", "run"))
```

Comment:
FYI. You could simplify this further:

```suggestion
    depends_on("py-numpy@1.15:", type=("build", "run"))
```